### DISCUSSION TITLE
QInput component behavior is not proper.

### DISCUSSION THREAD

**Discussion by harshalkumar-ishi** [quasarframework/quasar#16266] [General - Components / Directives / etc]
Implementing Edit form for a application using Quasar Components.
Taken a QInput component for input fields in forms data is showing for the specific form but there is an issue im facing for QInput type="number" field.

If i try to delete values for that number field it is not deleting or removing means i want to set that field as blank after clearing the values  inside.
In this case I'm not able to give validation for that number field.

Is this the correct behavior ?.

**Providing my code as follows in case im missing something just ping me:**

import JustPy as jp

def ch_submit_btn(self, msg):
    ...
    
async def input_demo_model1(request):
    wp = jp.QuasarPage()
    input_classes = "m-2 bg-gray-200 border-2 border-black-200 rounded xtw-64 py-2 px-4 text-gray-700 focus:outline-none focus:bg-white focus:border-purple-500"

    form = jp.QForm(a=wp, classes='q-gutter-md p-2 q-pl-xs')
    div = jp.QDiv(classes=input_classes, color="green", a=form)
    input = jp.QInput(a=div, type='number', classes=input_classes, placeholder='Please type here')
    btn_div = jp.QDiv(a=form, classes='q-gutter-y-md')
    submit_btn = jp.QBtn(
        a=btn_div, 
        label='Submit', 
        type='submit', 
        color='primary', 
        click= ch_submit_btn
        )
    return wp

jp.justpy(input_demo_model1)

**Comment by rstoenescu** [comment]
Can't tell the exact specifics, but are you binding a `v-model.number="yourVarFromCompScope"` to your QInput type=number? Notice `.number` there.

**Reply by harshalkumar-ishi** [reply]
No.
